src/07-projects/project-items.apib

```diff
@@ -78,6 +78,10 @@ Get a list of project items.
         + meta (object)
             + page (Pagination)
             + matches: `120` (number)
+            + totals (object)
+                + revenue (Money)
+                + costs (Money)
+                + result (Money)
```

Comment:
What does `result` mean?

Reply:
Should the `meta` return the sum of `internal_cost` and `billable_price` like this?

```
totals:
    internal_cost: ...
    billable_price: ...
```

So that it's obvious of what exactly the totals is the sum of.